modal/functions.py

```diff
@@ -1410,7 +1433,13 @@ async def spawn(self, *args: P.args, **kwargs: P.kwargs) -> "_FunctionCall[Retur
         if self._is_generator:
             invocation = await self._call_generator_nowait(args, kwargs)
         else:
-            invocation = await self._call_function_nowait(args, kwargs)
+            function_call_invocation_type = (
```

Comment:
I don't think anyone is using this in production, and it's an experimental flag we only shared with one user, so we can drop support for the `config` environment variable now. That feels more awkward than using `_experimental_spawn` directly.